src/cpp/xmlparser/XMLProfileManager.cpp

```diff
@@ -104,6 +107,19 @@ void XMLProfileManager::getDefaultTopicAttributes(TopicAttributes& topic_attribu
 
 XMLP_ret XMLProfileManager::loadDefaultXMLFile()
 {
+#ifdef _WIN32
+	char file_path[MAX_PATH];
+	size_t size = MAX_PATH;
+	if (getenv_s(&size, file_path, size, DEFAULT_FASTRTPS_ENV_VARIABLE) == 0 && size > 0)
```

Comment:
Remove return value of this function, Anyone is getting it.
Also print errors each time a loadXMLFile fails in this function.

Reply:
I have changed the returned value to void, and the errors are already logged by the `loadXMLFile` function.